### PR TITLE
javascript3-week1/srushti

### PR DESCRIPTION
**Pull request by srushtisalke**
Finished my javascript3 week1 homework
Please Review it.

### CODE REVIEW

**Comment by srushtisalke** [comment]
Hi @shpomp,
I am missing PR on this submission. Can you ask someone to review it? 

BR Srushti

File: 3_javascript/javascript3/week1/WeatherApp/script.js [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    } else if (temperature > 15 && temperature <= 30) {
+      return "fas fa-sun";
+    } else {
+      return "fas fa-fire";
+    }
+  }
+
+  function windInfo(data) {
+    const windInfo = document.createElement("li");
+    windInfo.innerText = "🌬️ " + data.wind.speed.toFixed(1) + " m/s";
```
Comment (ddobby94) [inline 3_javascript/javascript3/week1/WeatherApp/script.js:74]: When we are concatenating strings we usually use this syntax: `${variable} rest of the string.`

File: 3_javascript/javascript3/week1/WeatherApp/weather_api.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const weather_api = "https://api.openweathermap.org/data/2.5/weather?q=copenhagen&appid=d343416b21414202682daff848a54ebe";
+
+async function getWeather() {
```
Comment (ddobby94) [inline 3_javascript/javascript3/week1/WeatherApp/weather_api.js:3]: It seems like we are not using this function and this file for anything, so you could just delete it.

File: 3_javascript/javascript3/week1/WeatherApp/weather_city.js [hunk 3]
```diff
@@ -0,0 +1,71 @@
+
```
Comment (ddobby94) [inline 3_javascript/javascript3/week1/WeatherApp/weather_city.js:1]: It seems like this file is not being used, so we can just delete it

File: 3_javascript/javascript3/week1/WeatherApp/weather_city.js [hunk 4]
```diff
@@ -0,0 +1,71 @@
...
+  // the current search, make the slate blank using this snippet. 
+  resultsContainer.innerHTML = "";
+
+
+  const city = document.createElement("h2");
+  city.textContent = weather.name;
+  resultsContainer.append(city);
+
+  const icon = document.createElement("p");
+  icon.textContent = "Icon for the weather type: "  + weather.weather[0].icon;
```
Comment (ddobby94) [inline 3_javascript/javascript3/week1/WeatherApp/weather_city.js:52]: When we are concatenating strings we usually use this syntax: 
``` 
`${variable} rest of the string`
```

Comment (ddobby94) [reply 3_javascript/javascript3/week1/WeatherApp/weather_city.js:52]:
```suggestion
  icon.textContent = `Icon for the weather type: ${weather.weather[0].icon}`;
```

File: 3_javascript/javascript3/week1/WeatherApp/index.html [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+  </head>
+  <body>
+    <div class="container">
+      <h1 id="title">The weather in your city</h1>
+      <input type="text" id="nameCity" placeholder="Enter your city..." />
+      <button id="searchButtom">Search</button>
+      <ul id="weatherInfo"></ul>
+    </div>
+  </body>
+  <script src="weather_city.js"></script>
```
Comment (ddobby94) [inline 3_javascript/javascript3/week1/WeatherApp/index.html:22]: We can remove these `.js` file imports, since these are not being used.

**Review by ddobby94 — COMMENTED** [review]
Hi,

I'm David Dobiasz a new mentor in HYF. If you have any question regarding any of my comments feel free to reach out to me on Slack, I am very happy to talk about them! Programming can be difficult and overwhelming with all the little things at first... So really reach out to me if you are in doubt about anything.

It was nice reviewing your Homework, you did a good job! I only had comments about some smaller things. :)

**Comment by srushtisalke** [comment]
changes made in the base of review.

**Pull request closed without merging**